spec/services/workflow_errors_reporter_spec.rb

```diff
@@ -4,58 +4,20 @@
   let(:full_url) do
```

Comment:
I would move these "let" statements to be variables inside the test, since they are only used in that one place now ...  YMMV